src/plugin/physics/src/component/BoxCollider3D.hpp

```diff
@@ -0,0 +1,16 @@
+#pragma once
+
+#include <glm/glm.hpp>
+
+namespace ES::Plugin::Physics::Component {
+
+/**
+ * Basic collider used to check collisions through DetectABABCollisions system
```

Comment:
You can add the function referenced via the `@see` attribute in Doxygen

```suggestion
 * Basic collider used to check collisions through DetectABABCollisions system
 * @see System::DetectABABCollisions
```

Reply:
done